charts/astronomer/templates/houston/helm-hooks/houston-db-migration-job.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
   annotations:
     # This is what defines this resource as a hook. Without this line, the
     # job is considered part of the release.
-    "helm.sh/hook": pre-upgrade,post-upgrade,post-install
+    "helm.sh/hook": pre-upgrade,post-install
```

Comment:
the annotation is different here https://github.com/astronomer/astronomer/pull/2123/files

Reply:
https://github.com/astronomer/astronomer/blob/v0.33.3/charts/astronomer/templates/houston/helm-hooks/houston-db-migration-job.yaml#L17 - this is the original annotations

Reply:
this job is going to run after the upgrade as a seperate job and we are removing the annotation not to run on houston upgrade

Reply:
got it